### DISCUSSION TITLE
Compiling `wezterm` for CPU only system

### DISCUSSION THREAD

**Discussion by surmish** [wezterm/wezterm#6525] [General]
Is there a way to compile `wezterm` to avoid `OpenGL` driver requirements? I understand on bringing up `wezterm`, the `front_end=Software` option may be used. Is that the only way to ensure CPU-only usage? Does any GPU related requirement show up during compile?

**Comment by wez** [comment]
No, and that is not something that I plan to support.

You don't need a GPU to compile wezterm, but you do need something on your system that can translate from either OpenGL or WebGPU into something that can show on screen: that is provided by your system either via Mesa (and its software renderers) or your GPU drivers (for hardware).
